Dockerfile.konflux

```diff
@@ -1,9 +1,9 @@
-
 # Build arguments
 ARG SOURCE_CODE=.
 ARG VERSION
 
-FROM registry.redhat.io/ubi8/ubi-minimal@sha256:33161cf5ec11ea13bfe60cad64f56a3aa4d893852e8ec44b2fd2a6b40cc38539 AS stage
+# Base builder
+FROM registry.redhat.io/ubi9-minimal:9.5@sha256:30bbd445046a3a63f5f5557a3c67dee74e3c8e7855eb0347630b020f3689823f AS stage
```

Comment:
This concerns me @spolti

Reply:
@spolti If you are saying The base image is still on ubi8 then I have no objection to it.